.circleci/config.yml

```
@@ -632,6 +632,7 @@ workflows:
           filters:
             branches:
               ignore: master
```

Can we simplify this with a list of `ignore`s?

```suggestion
              ignore:
                # Do not spin up calypso.live for `master`
                - master
                # Do not spin up calypso.live for fork pull requests. Calypso.live will not build them.
                - /pull\/[0-9]+/
```

https://circleci.com/docs/2.0/configuration-reference/#branches-2